src/caselawclient/Client.py

```diff
@@ -150,6 +151,7 @@ def POST(
 
     def get_judgment_xml(self, judgment_uri, version_uri=None, show_unpublished=False) -> str:
         uri = f"/{judgment_uri.lstrip('/')}.xml"
+        show_unpublished = self.verify_show_unpublished(show_unpublished)
```

Comment:
I find the logic here really complicated (especially the type conversions); I don't know whether a specifically string-emitting version of this function would be useful, rather than handling it in the vars.
A bit of me doesn't like the shadowing of the "please show unpublished" by "you should show unpublished, although not having that potentially misleading variable hanging around is good.